src/collection/components/CollectionOrBundleEdit.tsx

```diff
@@ -195,6 +195,12 @@ const CollectionOrBundleEdit: FunctionComponent<
 		icon: IconName.alertTriangle,
 	} as LoadingInfo;
 
+	useEffect(() => {
+		setLoadingInfo({ state: 'loading' });
+		updateCollectionEditor();
+		setLoadingInfo({ state: 'loaded' });
+	}, []);
```

Comment:
preferably we use a react-query wrapper for this, so we get the loading state from the library for free.
eg: https://github.com/viaacode/hetarchief-client/blob/master/src/modules/visits/hooks/update-visit.ts